CRM/Event/BAO/Event.php

```diff
@@ -394,7 +394,7 @@ public static function getEventSummary() {
       $event_summary_limit = "LIMIT      0, $show_max_events";
     }
     else {
-      $event_summary_limit = "";
+      $event_summary_limit = '';
```

Comment:
I kept this nfc change in as it is the one place in the code where the setting is used so it kinda highlights it